app/helpers/date_and_time_helper.rb

```diff
@@ -0,0 +1,34 @@
+module DateAndTimeHelper
+  class << self
+    MINUTES_IN_AN_HOUR = 60
+
+    def display_time(minutes)
+      return nil if minutes.blank?
+
+      full_hour?(minutes) ? display_time_in_hours(minutes) : display_time_in_hours_and_minutes(minutes)
+    end
```

Comment:
I've replaced the guard clause syntax with a ternary since there was only one action following that clause.